### ISSUE TITLE
Can I use this image in 2019?

### ISSUE THREAD

**Issue by kadnan** [steveny2k/docker-predictionio#16]
Just wondering whether I can use this image for learning purpose(after updating latest versions)?

**Comment by steveny2k** [comment]
Yes, it is a good learning tool, even with the docker image in this project.
FYI, the official site provides the latest docker image.

**Comment by kadnan** [comment]
Yes, I need for learning purpose only.
